pom.xml

```diff
@@ -433,6 +432,11 @@
       <version>1.3.0</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>io.tabular</groupId>
+      <artifactId>tabular-client-runtime</artifactId>
```

Comment:
I think in some version, Iceberg includes everything so we don't even need tabular, can you look at docs or ask Tabular to see if this is true?

Reply:
Oh I see the Slack thread at https://affirm-ext.slack.com/archives/C04RZFH2494/p1687827154956799

Reply:
For Iceberg 1.3.0, its not required, so it should not be needed, but every time I omitted the AWS bundle, it wouldn't work without the `tabular-client-runtime`. TBH - there's a little more to unpack here, but figured I wanted to get something out that was slimmer and then we can further optimize later when needed